scripts/README.md

```
@@ -137,13 +137,24 @@ Use the PowerShell wrapper for convenience:
 
 ## import_transactions.py
 
-Upload a local transaction export to the running backend for parsing:
+Upload a local transaction export to the running backend for parsing and persistence:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)